### PR TITLE
Bump `gc-arena` to current master, take 2

### PR DESCRIPTION
**Pull request by moulins**
This required small changes to some Debug impls that were missed in #8964.

### CODE REVIEW

File: core/Cargo.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ byteorder = "1.4"
 bitstream-io = "1.6.0"
 flate2 = "1.0.25"
 fnv = "1.0.7"
-gc-arena = { git = "https://github.com/ruffle-rs/gc-arena" }
+gc-arena = { git = "https://github.com/ruffle-rs/gc-arena", rev = "3195fbed9f2ba1683fb113e3d7d84794fe3465d5" }
```
Comment (danielhjacobs) [inline core/Cargo.toml:15]: Am I missing something or shouldn't this be kyren/gc-arena now?

Comment (Aaron1011) [reply core/Cargo.toml:15]: You're right - we no longer need our fork.

Comment (moulins) [reply core/Cargo.toml:15]: Oops, right, good catch :c

File: render/wgpu/Cargo.toml [hunk 2]
```diff
@@ -21,7 +21,7 @@ image = { version = "0.24.5", default-features = false }
 ouroboros = "0.15.5"
 typed-arena = "2.0.1"
 once_cell = "1.17.0"
-gc-arena = { git = "https://github.com/ruffle-rs/gc-arena" }
+gc-arena = { git = "https://github.com/kyren/gc-arena", rev = "3195fbed9f2ba1683fb113e3d7d84794fe3465d5" }
```
Comment (Aaron1011) [inline render/wgpu/Cargo.toml:24]: Can you make this a workspace dependency:
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#inheriting-a-dependency-from-a-workspace

**Review by Aaron1011 — CHANGES_REQUESTED** [review]

File: render/wgpu/Cargo.toml (hunk 2 above)
Comment (moulins) [reply render/wgpu/Cargo.toml:24]: TIL, I didn't know that was possible.

Comment (Aaron1011) [reply render/wgpu/Cargo.toml:24]: It's a pretty new feature, but we're using the latest stable Rust, so we can use it :)

**Comment by Lord-McSweeney** [comment]
There are a lot of `Invalid session id` errors in the test results.

**Review by Aaron1011 — APPROVED** [review]

**Pull request merged**